flatdata-generator/tests/generators/rust_expectations/archives/comments.rs

```diff
@@ -0,0 +1,232 @@
+// Do not edit: This code was generated by flatdata's generator.
```

Comment:
remove all non-relevant code (e.g. the schemas?)

Reply:
Done.